bindings/wasm/examples-account/src/config.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {AccountBuilder, Client, Network, ExplorerUrl, Config, DIDMessageEncoding, AutoSave } from './../../node/identity_wasm.js';
```

Comment:
It's possible, Although it defeats the purpose of making the examples very simple, since this PR if focused on the bindings. But the examples are likely to move in a future PR.